bridge/scripts/tasks/alicenetTasks.ts

```diff
@@ -1750,6 +1751,251 @@ task("update-alicenet-node-version", "Set the Canonical AliceNet Node Version")
     console.log("Done");
   });
 
+task(
+  "atomic-mintTo-publicStaking",
+  "mints a public staking position to 1 recipient with alca token amount from alicenet factory and locked for max staking lock duration"
+)
+  .addParam("recipient", "address to minto staked position to")
+  .addParam("alcaAmount", "amount of ALCA to send to staked position")
+  .addFlag(
+    "test",
+    "test mode for use with a hardhat fork mode on, you must uncomment forking section in hardhat config, add ALCHEMY_ENDPOINT_ETH to .env file with alchemy endpoint"
+  )
+  .addOptionalParam(
+    "waitConfirmation",
+    "wait specified number of blocks between transactions",
+    0,
+    types.int
+  )
+  .setAction(async (taskArgs, hre) => {
+    const waitConfirmationsBlocks = await parseWaitConfirmationInterval(
+      taskArgs.waitConfirmation,
+      hre
+    );
+    let factory = await hre.ethers.getContractAt(
+      ALICENET_FACTORY,
+      "0x4b6dF6B299fB6414f45719E0d9e1889269a7843E"
```

Comment:
Can we get this as parameter and use the factory to retrieve the rest. This can make this more testable.